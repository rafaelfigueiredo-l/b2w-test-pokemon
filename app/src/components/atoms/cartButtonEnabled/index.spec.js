import CartButtonEnabled from "./";

describe("<CartButtonEnabled />", () => {
  const image = require("../../../assets/images/pokemonImageNotFound.png");

  const component = <CartButtonEnabled>Texto</CartButtonEnabled>;

  const mounted = mount(component);
  const wrapper = shallow(component);

  it("should mount correctly CartButtonEnabled component", () => {
    expect(wrapper.find(".cartButtonEnabled")).toHaveLength(1);
    expect(mounted).toMatchSnapshot();
  });

  it("should unmount correctly CartButtonEnabled component", () => {
    expect(mounted.unmount()).toEqual({});
  });

  it("should have a children", () => {
    expect(wrapper.find(".cartButtonEnabled").children()).toHaveLength(1);
  });
});
