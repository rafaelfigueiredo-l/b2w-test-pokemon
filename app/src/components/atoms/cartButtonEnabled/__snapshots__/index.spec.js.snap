// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CartButtonEnabled /> should mount correctly CartButtonEnabled component 1`] = `
<styled.button
  className="cartButtonEnabled"
>
  <button
    className="sc-AxjAm xhULX cartButtonEnabled"
  >
    Texto
  </button>
</styled.button>
`;
